docs/Authenticating-Users-via-keycloak-SAML.md

```diff
@@ -0,0 +1,135 @@
+# Introduction
```

Comment:
Please update the documentation to the version you sent me in Slack.  And add a note to that version to set the "SAML Attribute Name" to "email" to match your screenshot in the "X500 Email" email page.